translations/en-us.yaml

```diff
@@ -261,6 +261,7 @@ general:
   permissionsInOtherSchools: "Permissions in Other Schools"
   perPage: "Per Page"
   phone: Phone
+  pleaseSelect: "Please Select"
```

Comment:
I like the generic, but everywhere else we have `Select School` or `Select Course` I think this one should be `Select Year`.

Reply:
Fixed.